website/docs/add-secure-apps/flows-stages/flow/context/index.mdx

```diff
@@ -62,7 +64,7 @@ When an unauthenticated user attempts to access a secured resource, they are red
 
 When a user authenticates/enrolls via an external source, this will be set to the source they are using.
 
-#### `outpost` (dictionary) <span class="badge badge--version">authentik 2024.10+</span>
+#### `outpost` (dictionary)&nbsp;<VersionBadge semver="2024.10" />
```

Comment:
Same with this, I'm assuming there's no simple way of doing this in a cleaner way without using &nbsp;